sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/CHANGELOG.md

```diff
@@ -1,15 +1,11 @@
 # Release History
 
-## 1.0.0-beta.7 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 1.0.0-beta.7 (2022-08-02)
```

Comment:
Please do not change the date here, this value will be updated before we truly release it.